packages/vuetify/src/components/VBanner/VBanner.tsx

```diff
@@ -54,24 +57,31 @@ export const VBanner = defineComponent({
     const { elevationClasses } = useElevation(props)
     const { positionClasses, positionStyles } = usePosition(props)
     const { roundedClasses } = useRounded(props)
+    const lineClasses = computed(() => props.lines ? `v-banner--${props.lines}-line` : undefined)
+
+    provideDefaults({
+      VBannerActions: {
+        color: toRef(props, 'color'),
+        density: toRef(props, 'density'),
+      },
+    })
```

Comment:
You're already rendering this component, just pass the props directly?

Reply:
Parent components that target children should do so with a blanket provideDefaults or in some cases, wrapping a specific slot. This enables the user to recreate the component in multiple ways without having to do manual things.

If I add the props manually, I have to pass them down manually; which means the user has to manually do something.